sdk/template/template/package.json

```diff
@@ -1,14 +1,24 @@
 {
   "name": "@azure/template",
-  "version": "1.0.10-beta.1",
-  "description": "Template Library with typescript type definitions for node.js and browser.",
+  "version": "1.0.11-beta.1",
+  "description": "Example project for learning how to build a client library",
   "sdk-type": "client",
   "main": "dist/index.js",
   "module": "dist-esm/src/index.js",
-  "browser": {
-    "./dist-esm/src/print.js": "./dist-esm/src/print.browser.js"
-  },
+  "browser": {},
```

Comment:
demonstrating this can be helpful too as an advanced topic maybe?

Reply:
I think it can be helpful, but the old example wasn't particularly authentic and there's no really good opportunity to do it in this client either. Maybe @xirzec or @chradek will have an idea for how we can work browser mapping into this app config client in an authentic way.

Reply:
The only thing in the current appconfig is some crypto helpers which should be obsolete soon enough. I think revisiting this when we have a good demonstrative scenario is fine.

Reply:
can we open an issue to track this?